up42/storage.py

```diff
@@ -220,12 +220,12 @@ def get_orders(
             "status": set(statuses) & allowed_statuses if statuses else None,
         }
         params = {k: v for k, v in params.items() if v is not None}
-
-        if statuses is not None and len(statuses) > len(params["status"]):
```

Comment:
This whole block is not needed, let's just drop it

Reply:
do we not want to add logs when statuses is not included in allowed_stasuses?

Reply:
I don't see any reason for that - if a user uses an IDE, then type hints will show incorrect statuses, if not, then they need to rely on documentation. Also this does not create any error